icechunk-python/Cargo.toml

```diff
@@ -24,14 +24,18 @@ futures = "0.3.31"
 icechunk = { path = "../icechunk", version = "0.1.0-alpha.7" }
 pyo3 = { version = "0.23", features = [
   "chrono",
-  "extension-module",
+  #  do we really need this one?
+  #  "extension-module",
```

Comment:
Yes I think so, see https://pyo3.rs/v0.23.3/features#extension-module

Reply:
Yeah, maybe, I don't think we are building a python extension module though? Maybe we give it a try without? Who knows :shrug: